package.json

```diff
@@ -40,11 +40,7 @@
     "lint-staged": "^10.5.3",
     "prettier": "2.2.1"
   },
-  "husky": {
```

Comment:
why did you remove the pre-commit?

Reply:
it not allow me to push

Reply:
Do not remove husky, this will run lint before you push your code. Please refer to eslint and husky libraries and read about them. 
- Additionally, revert these